### PR TITLE
Update free-courses-pt_BR.md

### PR DESCRIPTION
**Pull request by guilhermej**
Add free course Python básico from Solyd (https://solyd.com.br/treinamentos/python-basico/)

## What does this PR do?
Add resource

### Description 
Add free course Python básico from Solyd (https://solyd.com.br/treinamentos/python-basico/)

### Why is this valuable (or not)?
Add a new free course

### How do we know it's really free?
You can check it on the website https://solyd.com.br/treinamentos/python-basico/

### For book lists, is it a book? For course lists, is it a course? etc.
It's an online video course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists  in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

**Comment by eshellman** [comment]
Don't invent titles! Use the title given by the resource.

**Comment by heonyounglee** [comment]
ㅇㄴㅁㅇㄹㄴㄹ

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -257,6 +257,7 @@
 * [Programação em Python: O Guia para Iniciantes](https://www.udemy.com/course/programacao-python-guia-para-iniciantes/) - Jones Granatyr (Udemy)
 * [Python 3 na Prática](https://www.udemy.com/python-3-na-pratica/) - João Batista (Udemy)
 * [Python 3 na Web com Django (Básico e Intermediário)](https://www.udemy.com/python-3-na-web-com-django-basico-intermediario/) - Gileno Alves Santa Cruz Filho (Udemy)
+* [Python Básico (Com certificado)](https://solyd.com.br/treinamentos/python-basico) - Guilherme Junqueira (Solyd Offensive Security)
 * [Python Fundamentos para Análise de Dados](https://www.datascienceacademy.com.br/course?courseid=python-fundamentos) - Data Science Academy
 * [Python para Competições de Programação](https://www.youtube.com/playlist?list=PLMxflQ9_eOd9CY6Id5gfs3Edqt8vLC47p) - Adorilson (YouTube)
```
Comment (5jll) [inline courses/free-courses-pt_BR.md:262]:
```suggestion
 * [Python para Competições de Programação](https://www.youtube.com/playlist?list=PLMxflQ9_eOd9CY6Id5gfs3Edqt8vLC47p) - a3thary (YouTube)```

```

**Review by 5jll — COMMENTED** [review]
```suggestion
 * [Python para Competições de Programação](https://www.youtube.com/playlist?list=PLMxflQ9_eOd9CY6Id5gfs3Edqt8vLC47p) - a3thary (YouTube)

**Comment by 5jll** [comment]
> ```suggestion
>  * [Python para Competições de Programação](https://www.youtube.com/playlist?list=PLMxflQ9_eOd9CY6Id5gfs3Edqt8vLC47p) - dfgn (YouTube)

**Review by 5jll — APPROVED** [review]

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -257,6 +257,7 @@
 * [Programação em Python: O Guia para Iniciantes](https://www.udemy.com/course/programacao-python-guia-para-iniciantes/) - Jones Granatyr (Udemy)
 * [Python 3 na Prática](https://www.udemy.com/python-3-na-pratica/) - João Batista (Udemy)
 * [Python 3 na Web com Django (Básico e Intermediário)](https://www.udemy.com/python-3-na-web-com-django-basico-intermediario/) - Gileno Alves Santa Cruz Filho (Udemy)
+* [Python Básico (Com certificado)](https://solyd.com.br/treinamentos/python-basico) - Guilherme Junqueira (Solyd Offensive Security)
```
[the remark this answers is not in the document]
Comment (SethFalco) [reply courses/free-courses-pt_BR.md:260]: Closing suggestion because it's just spam.

Comment (SethFalco) [inline courses/free-courses-pt_BR.md:260]:
```suggestion
* [Python Básico](https://solyd.com.br/treinamentos/python-basico) - Guilherme Junqueira (Solyd Offensive Security)
```

**Pull request merged**